lines/client/__init__.py

```diff
@@ -1,6 +1,6 @@
 """BetFund client.client namespace."""
-from .rundown import RundownClient
+from .rundown import Rundown
 from .config import RundownSportId
 
 
-__all__ = ["RundownClient", "RundownSportId"]
+__all__ = ["Rundown", "RundownSportId"]
```

Comment:
Why not -
```
__all__ = [Rundown, RundownSportId]
```

Reply:
Same for many of the `init`'s below.

Reply:
https://docs.python.org/3/tutorial/modules.html#importing-from-a-package